sdk/storage/inc/datalake/datalake_utilities.hpp

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include "common/crypt.hpp"
+
+#include <map>
+#include <string>
+
+namespace Azure { namespace Storage { namespace DataLake {
+
+  inline std::map<std::string, std::string> DeSerializeMetadata(
```

Comment:
should we use `DeSerialize` or `Deserialize`?

Reply:
Some further research shows that `deserialize` is an actual word so I'll change to `Deserialize`. Ref: https://www.yourdictionary.com/deserialize#:~:text=deserialize,to%20reconstruct%20the%20original%20object.

Reply:
Also, `de` is not a word, sorry Victor.